lib/slackify.rb

```diff
@@ -63,6 +63,7 @@ def payload
               color: color,
               text: text,
               fields: fields,
+              mrkdwn_in: [fetch(:slack_mrkdwn_in)].flatten,
```

Comment:
Given that the default is an array, could we do without `[].flatten` here and just `fetch(:slack_mrkdwn_id)` ?